packages/lesswrong/lib/vulcan-lib/routes.ts

```diff
@@ -13,7 +13,7 @@ export type RouterLocation = {
   url: string,
   hash: string,
   params: Record<string,string>,
-  query: Record<string,string>,
+  query: Record<string,string>, // TODO: this should be Record<string,string|string[]>
```

Comment:
Why couldn't you change this?

Reply:
Changing it resulted in ~70 ts errors, which I didn't want to fix in this PR